magento/magento2
# Add "Last Updated At" Column to Admin Product Grid

# Description
When managing products in the Adobe Commerce admin, it would be useful to see when they were last updated. This pull request adds that information to the grid. The information already exists in the database.

# Manual testing scenarios
1. Log into the admin
2. Navigate to `Catalog -> Products`
3. Notice the list of products lacks information about the last time it was updated
4. Apply the changes in this pull request
5. Notice the list of products now shows the date/time it was last updated

# Contribution checklist

### Resolved issues:
1. [x] resolves magento/magento2#35488: Add "Last Updated At" Column to Admin Product Grid
